app/models/va_health_facility.rb

```diff
@@ -0,0 +1,116 @@
+# frozen_string_literal: true
+require_dependency 'facilities/client'
+
+class VAHealthFacility < ActiveModelSerializers::Model
+  attr_accessor :station_id, :station_number, :visn_id, :name, :classification, :lat, :long,
+                :address, :phone, :hours, :services
+
+  def self.query(bbox:, services:)
+    results = client.query(bbox: bbox.join(','),
+                           where: VAHealthFacility.where_clause(services))
+    results.each_with_object([]) do |record, facs|
+      facs << VAHealthFacility.from_gis(record)
+    end
+  end
+
+  def self.find_by_id(id:)
+    results = client.get(identifier: id)
+    VAHealthFacility.from_gis(results.first) unless results.blank?
```

Comment:
So when you do #find_by_<something> you normally would not have named arguments, instead just a single argument. Whereas self.find_by(id: ) does make sense. But this is just a nit-pick